omero/sysadmins/version-requirements.rst

```diff
@@ -141,107 +148,111 @@ Components
        :header-rows: 1
 
        * - Python
-         - OMERO.web 5.3
-         - OMERO.server 5.3
          - OMERO.web 5.4
          - OMERO.server 5.4
          - OMERO 5.5
+         - OMERO 5.6
        * - 2.6
-         - |Broken|
-         - |Supported|
          - |Broken|
          - |Supported|
          - |Dropped|
+         - |Dropped|
        * - 2.7
          - |Recommended|
          - |Recommended|
          - |Recommended|
          - |Recommended|
-         - |Recommended|
        * - 3.x
          - |Broken|
          - |Broken|
          - |Broken|
          - |Broken|
```

Comment:
If we are still using this page to indicate our provisional plans for the next major release of OMERO, Python 3.x support is undoubtedly the next large body of work in terms of requirements.

Reply:
It is. I didn't dare hope that we expect to support it for OMERO 5.6 but perhaps I ought to be more optimistic?

Reply:
I think we have to be optimistic about getting to Python 3 at this point.